opencensus/log/__init__.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
+from copy import copy
+import logging
+
+from opencensus.trace import execution_context
+
+
+_meta_logger = logging.getLogger(__name__)
+
+TRACE_ID_KEY = 'traceId'
+SPAN_ID_KEY = 'spanId'
+SAMPLING_DECISION_KEY = 'traceSampled'
+
+LogAttrs = namedtuple('LogAttrs', ['trace_id', 'span_id', 'sampling_decision'])
+ATTR_DEFAULTS = LogAttrs('', '', False)
```

Comment:
Would it be better to use the invalid trace and span IDs as defaults?  The Log4j version does that, though I don't know if it makes sense here.

https://github.com/census-instrumentation/opencensus-java/blob/b552db4432d5cc1a4767f49ed8da114631d06b99/contrib/log_correlation/log4j2/src/test/java/io/opencensus/contrib/logcorrelation/log4j2/OpenCensusTraceContextDataInjectorTest.java#L92-L108

Reply:
That seems like good default behavior, and we may as well keep this consistent between clients. Changed in 7fa82f2.